src/Chainweb/Chainweb.hs

```diff
@@ -674,7 +690,7 @@ runChainweb cw = do
 
     -- collect server resources
     proj :: forall a . (ChainResources logger -> a) -> [(ChainId, a)]
-    proj f = map (second f) chains
+    proj f = chains & mapped . _2 %~ f
```

Comment:
Nit:

this change seems extraneous, is lens the preferred style?
personally I find the old version more readable

Reply:
The old version is actually the preferred style in our code. We don't tend to use the Lens operators as much.